src/MTProtoTools/UpdateHandler.php

```diff
@@ -619,6 +620,10 @@ public function saveUpdate(array $update): void
         if (isset($update['message']['peer_id'])) {
             $update['message']['to_id'] = $update['message']['peer_id'];
         }
+        if ($this->settings->getDb()->getEnableUpdatesDb()) {
+            $this->updates_db[(int)(microtime(true)*1000*1000)] = $update;
```

Comment:
Let's not do this please, this is not needed, you can just use getUpdates instead

Reply:
You mean you dont want to store updates in database? Its very common request from users. Most users dont need event handlers, they just need updates in db and work with them as they want :)

But if you dont want to store it then we can just drop second commit. 

Also can you pls update examples in docs for event handler database?

Reply:
I dont get it. $this->updates is just array, not dbArray. It will be cleared on restart?

Anyway webhook and getUpdates should be enought. Thanks!

Reply:
`$this->updates` is never cleared on restart to avoid loosing updates in certain conditions.